docs/data/charts/axis/axis.md

```diff
@@ -218,6 +218,15 @@ To avoid overlapping, you can use the `height` prop for `xAxis` and `width` for
 
 {{"demo": "MultipleAxes.js"}}
 
+### Overlapping axes
+
+In order to overlap axes, the axis size can be set to `0`.
```

Comment:
This feels a bit like a workaround. [In this PR](https://github.com/mui/mui-x/pull/16709) we're going to start taking the size into account and it will cause the labels to be hidden:

![image](https://github.com/user-attachments/assets/c77ecbd5-a54f-476f-8921-a39b0d7cbff4)

Should we consider a less hacky API?

Reply:
We can also just drop this if it is too much. Translating the axis will still make the axis count into the sizes though

Reply:
I see some value in this example, so it might make sense to have it in the docs. However, we should probably use a custom component for it, or another workaround, since this one will break in the short term.